yellowbox/image_build.py

```diff
@@ -17,6 +19,56 @@ class DockerfileParseError(BuildError):
 DockerfileParseException = DockerfileParseError  # legacy alias
 
 
+def _docker_build(docker_client: DockerClient, file: TextIO, async_run: bool, **kwargs):
```

Comment:
I don't like the async_run parameter, why not have the async function call this helper function with a StringIO and only print it on failure?